ecosystem/sep-0010.md

```diff
@@ -22,19 +22,21 @@ The authentication flow is as follows:
 
 1. The client obtains a unique [`challenge`](#challenge), which is represented as specially formed Stellar transaction
 1. The client verifies that the transaction has an invalid sequence number 0.  This is extremely important to ensure the transaction isn't malicious.
-1. The client signs the transaction using the secret key for the user's Stellar account
+1. The client signs the transaction using the secret key for the user's key pair
 1. The client submits the signed challenge back to the server using [`token`](#token) endpoint
 1. If the signature checks out, the server responds with a [JWT](jwt.io) that represents the user's session
 1. Any future calls to the server can be authenticated by including the JWT as a parameter
 
 The flow achieves several things:
 
 * Both client and server part can be implemented using well-established Stellar libraries
-* The client can verify that the server holds the secret key to a particular account
-* The server can verify that the client holds the secret key to their account
+* The client can verify that the server holds the secret key to a particular public key
+* The server can verify that the client holds the secret key to a particular public key
 * The client is able to prove their identity using a Ledger or other hardware wallet as well as by having direct access to the secret key
 * The server can chose its own timeout for the user's session
 
+The result of the challenge-response and the JWT generated when implementing this SEP will confirm that an authenticating user has possession of a Stellar secret key but does not prove signing capabilities of any Stellar account. A server receiving the JWT that needs proof of control of a Stellar account should verify if the Stellar key authenticated in the JWT is a signer of the account and has a weight that is suitable for the applications operations and purpose. An application wanting to prove that the signer has complete authority of the account may want to verify that the weight of the signer is equal or greater than all thresholds on the account.
```

Comment:
This is good, maybe rephrase a little more simply?

"A server receiving the JWT that needs proof of control of a Stellar account should verify if the Stellar key authenticated in the JWT is a signer of the account and has a weight that is suitable for the applications operations and purpose."
->
In the case of a server that needs proof of control of a Stellar account, the server should verify that the Stellar key authenticated in the JWT has sufficient signing weight to meet the [threshold](https://www.stellar.org/developers/guides/concepts/multi-sig.html#thresholds) suitable for the application's purposes.